package.json

```diff
@@ -1,20 +1,47 @@
 {
   "name": "@peak-ai/ais-service-discovery",
-  "version": "0.2.0",
-  "main": "src/index.js",
+  "version": "0.3.0-rc1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "repository": "git@github.com:peak-ai/ais-service-discovery-js.git",
-  "author": "<ewan.valentine@peak.ai>",
+  "author": "<engineering@peak.ai>",
   "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn format && yarn preupdate"
+    }
+  },
   "scripts": {
-    "test": " jest ./src/**",
-    "lint": "eslint ./src"
+    "lint": "eslint jest.config*js 'src/**/*.ts'",
+    "format": "prettier --write jest.config*js 'src/**/*.ts'",
```

Comment:
Could add `.json` and `.js` into linting & prettier here. Think `src/**/*.{ts,js,json}` might work.

Reply:
I couldn't add `.js` because the linting's Typescript specific I think, so it threw loads of errors for the legacy stuff